website/docs/r/container_group.html.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_group"
+sidebar_current: "docs-azurerm-resource-container-group"
+description: |-
+  Create as an Azure Container Group instance.
+---
+
+# azurerm\_container\_group
+
+Create as an Azure Container Group instance.
+
+## Example Usage (Linux)
+
+```hcl
+resource "azurerm_resource_group" "aci-rg" {
+  name     = "aci-test"
+  location = "west us"
+}
+
+resource "azurerm_container_group" "aci-helloworld" {
+  
+  name = "aci-hw"
+  location = "west us"
+  resource_group_name = "${azurerm_resource_group.aci-rg.name}"
+  ip_address_type="public"
+  os_type = "linux"
+
+  container {
+        name = "hw"
+        image = "microsoft/aci-helloworld:latest"
+        cpu ="0.5"
+        memory =  "1.5"
+        port = "80"
+    }
+    container {
+        name = "sidecar"
+        image = "microsoft/aci-tutorial-sidecar"
+        cpu="0.5"
+        memory="1.5"
+    }
+
+  tags {
+    environment = "testing"
+  }
+}
+```
+
+## Example Usage (Windows)
+
+```hcl
+resource "azurerm_resource_group" "aci-rg" {
+  name     = "aci-test"
+  location = "west us"
+}
+
+resource "azurerm_container_group" "winapp" {
+  
+  name = "mywinapp"
+  location = "west us"

```

Comment:
_minor_ can we update this to be `"${azurerm_resource_group.aci-rg.location}"`

Reply:
Done.

Reply:
Removed the windows example.